src/dash/utils/TimelineSegmentsGetter.js

```diff
@@ -48,38 +48,17 @@ function TimelineSegmentsGetter(config, isDynamic) {
         }
     }
 
-    function getSegmentsFromTimeline(representation, requestedTime, index, availabilityUpperLimit) {
-        checkConfig();
```

Comment:
Not really removed, it is a "visual effect" of github diff tool. Both getSegmentByIndex and getSegmentByTime check both the config and the existence of the representation. I am not doing it in iterateSegments because this function is only called by the previous ones.

Reply:
Not really removed. It is a "visual effect" of the github diff tool. Both getSegmentByTime and getSegmentByIndex methods do the config and representation check. It was only removed from iterate method, but just because it a method called just by the two previous methods.